tests/Agent/IntegrationTests/IntegrationTests/RemoteServiceFixtures/ServiceStackApplicationFixture.cs

```diff
@@ -15,8 +16,8 @@ public ServiceStackApplicationFixture() : base(new RemoteWebApplication("Service
 
         public void GetHello()
         {
-            var address = $"http://{DestinationServerName}:{Port}/hello/Worked";
-            DownloadJsonAndAssertEqual(address, new HelloResponse("Worked"));
+            var address = $"http://{DestinationServerName}:{Port}/hello/Worked";//"?format=json";
```

Comment:
Is the comment leftover code from some older work in progress stuff?

Reply:
Yep, thanks for catching that.